electron/graph.js

```diff
@@ -122,19 +124,61 @@ function findOutliers(objs, whitelist, found = []) {
 	return found
 }
 
-function onNodeClicked(data) {
-	console.log('Clicked on node point with data:', data)
-
-	let outliers = findOutliers(data.branchset, getWhitelist())
-	console.log('Outliers found:', outliers)
+function ToggleMuteLeaves(doMute){
+	var nodes = document.querySelectorAll(".node.leaf")
+	nodesMuted = doMute
+	for(var i=0;i<nodes.length;i++){
+		var node = nodes[i];
+		var isMuted = node.className.baseVal.indexOf("muted") != -1;
+		if(doMute && !isMuted){
+			node.className.baseVal += " muted"
+		}
+		if(!doMute && isMuted){
+			node.className.baseVal = node.className.baseVal.replace("muted","")
+		}
+		
+	}
+}
 
-	outliers.forEach(outlier =>
-		document.getElementById(outlier).setAttribute('fill', 'green')
-	)
+function onNodeClicked(data) {
+	if(data.name == "" || nmResults == undefined)
+		return; // We don't care about anything that's not a leaf node
+	// Find the other individual that is nonmonophyletic with this one
+	var nonmono_pair;
```

Comment:
Please use camelCase to name variables, and please use `let/const` instead of `var`.